Server/controllers/leave.controller.js

```diff
@@ -1,18 +1,58 @@
 "use strict";
+const leavesDataLayer = require('../datalayer/leave.datalayer');
 /**
  * Class representing leaves controller
  *
  * @class leavesController
  */
 class leavesController {
 
-    static getLeavesByUserId (req, res) {}
+    static async getLeavesByUserId (req, res) {
+        const userId = req.query.user;
+        let leavesList;
```

Comment:
we can name is `leaves` instead of `leavesList`

Reply:
Renamed.